packages/cli/src/cmds/resource.ts

```diff
@@ -266,6 +267,59 @@ export class ResourceAddFunction extends YargsCommand {
   }
 }
 
+export class ResourceAddKeyVault extends YargsCommand {
+  public readonly commandHead = `azure-keyvault`;
+  public readonly command = `${this.commandHead}`;
+  public readonly description = "Add a new Azure Key Vault service.";
```

Comment:
@MuyangAmigo Pleas advise the description.
![image](https://user-images.githubusercontent.com/1050213/144567217-5bd8dc6e-7b5e-4c9c-abbf-d2ed03186f89.png)